pkg/executor/statefulset.go

```diff
@@ -429,7 +461,6 @@ func generateVolumeMounts(
 	nativeSystemdSupport bool,
 	persistDirs []string,
 ) ([]v1.VolumeMount, []v1.Volume, error) {
-
```

Comment:
I know you like to have a blank line at the beginning of each function. Did you intend to delete this one?